packages/genesys-spark-components/src/components/beta/gux-selector-cards/example.html

```diff
@@ -31,3 +31,62 @@ <h2>Simple Selector Card Group</h2>
     <gux-icon slot="icon" icon-name="fa/play-regular" decorative />
   </gux-selector-card-beta>
 </gux-selector-cards-beta>
+
+<h2>Descriptive Selector Card Group</h2>
+<gux-selector-cards-beta class="example2">
+  <gux-selector-card-beta variant="descriptive">
+    <label slot="label" for="first">First</label>
+    <input slot="input" id="first" type="radio" name="example2" />
+    <span slot="content"
```

Comment:
I think `description` might be a better slot name

Reply:
Done